src/scripts/push-docker-image.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+PARAM_HEROKU_API_KEY="${!PARAM_HEROKU_API_KEY}"
+
+if [ "$PARAM_RECURSIVE" == "1" ];then
+  set -- "$@" --resursive
+fi
+
+heroku container:push -a "$PARAM_APP_NAME" "$@" "$PARAM_PROCESS_TYPES"
```

Comment:
It's always nice to wrap dynamically populated commands in `set -x` and `set +x`. It allows users to see what is executed in runtime:

```
set -x
heroku container:push -a "$PARAM_APP_NAME" "$@" "$PARAM_PROCESS_TYPES"
set +x
```

Reply:
Very nice